### PR TITLE
Flesh out getting started locally details in README.md

### LINKED ISSUE #41: No documentation on getting the API working locally if you use Docker For Desktop instead of Docker Toolbox
Docker Toolbox is considered a "Legacy Solution" according to Docker and likely will go unsupported at some point as Docker has built out the Docker For WSL2 backend and WSL2 will be the current version of WSL for all users after the [Windows 10 2004 Update](https://devblogs.microsoft.com/commandline/wsl2-will-be-generally-available-in-windows-10-version-2004/) and is currently available for members of the [Windows Insider Program](https://insider.windows.com/en-us/welcome-back/) on the 'Release Preview'  or 'Fast Ring'

Docker for Desktop is the recommended way of working with Docker for Windows 10 Pro, Windows 10 Enterprise, Windows Server, and macOS and our instructions to not have details for getting the docker containers set up with Docker for Desktop or details for getting the Docker Virtual Machine spun up even with Docker for Toolbox. Spinning up a Docker Container is not something most Lambda Students will have the knowledge of as the Lambda Curriculum does not cover DevOps practices and Tools.

### PR DESCRIPTION
**Pull request by CodyBrewer**
# Description

- Adheres styling in *.md files in the repo to markdownlint standards
- Adds information on whether you should install Docker for Desktop or Docker Toolbox
- Adds details on getting docker containers running using [docker-compose](https://docs.docker.com/compose/)

Fixes # (41)

## Type of change
- [x] Bug fix (non-breaking change which fixes an **issue)**
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] I was able to get my local dev environment running by following the getting started section using Docker for Desktop 

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by CodyBrewer** [comment]
[Flesh out getting started locally documentation in README.md](https://trello.com/c/ArVVyD0K/31-flesh-out-getting-started-locally-documentation-in-readmemd)

**Review by CodyBrewer — COMMENTED** [review]
📖 Self-Review Completed

File: README.md [hunk 1]
```diff
@@ -1,100 +1,70 @@
...
-  
-
 To get the server running locally:
 
-  
+ -Prerequisites:
+  Install [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/) if you are running Windows 10 Home Edition pre-2004 upate.
 
-- Clone this repo.
+  Install Docker For Desktop if you are running [Windows 10 Home post-2004 update](https://docs.docker.com/docker-for-windows/install-windows-home/), [Pro, Enterprise or Education](https://docs.docker.com/docker-for-windows/install/), or [macOS](https://docs.docker.com/docker-for-mac/install/)
```
Comment (dave-vazquez) [inline README.md:16]: Awesome 🎉  Loved that you got a jump on updating the documentation. This will be a big help for our's and future teams 👍

**Review by dave-vazquez — APPROVED** [review]

**Review by ccsmith13 — APPROVED** [review]
looks good to me!

**Review by alesslongaretti — APPROVED** [review]

**Pull request merged**